player_test.py

```diff
@@ -0,0 +1,164 @@
+import unittest
+from player import Player
+
+class InitTest(unittest.TestCase):
+  def test___init___method_works(self):
+  # test 30
+  # This test was written rertrospectively; the __init__ method came about 
+  # naturally during refactoring of the code into the Player class.
+    mario = Player("placeholder name")
+    expected_mario_hand = {}
+    expected_mario_score = 0
+    self.assertIsInstance(mario, Player)
+    self.assertEqual(mario.hand, expected_mario_hand)
+    self.assertEqual(mario.score, expected_mario_score)
```

Comment:
As the name attribute is also set in the init method, it should be checked here too. So that all of the things that happen in the init are checked.